src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -141,11 +146,25 @@ def password_correct?
         end
 
         # Password is considered strong when cracklib returns an empty message.
+        # User has the last word to decide whether to use a weak password.
         def password_strong?
           message = cracklib_message
           return true if message.empty?
-          Yast::Report.Warning(message)
-          false
+
+          popup_text = [
+            _("The password is too simple:"),
```

Comment:
I am not sure if problem is always too simple, as it can have different problem. What about "The password is too weak"? which fits better?